packages/nuxt/src/app/plugins/check-if-layout-used.ts

```diff
@@ -16,7 +17,10 @@ export default defineNuxtPlugin({
       }
     }
     if (import.meta.server) {
-      nuxtApp.hook('app:rendered', checkIfLayoutUsed)
+      nuxtApp.hook('app:rendered', async () => {
+        await nextTick()
+        checkIfLayoutUsed()
+      })
```

Comment:
there might be some value in not awaiting the nextTick as that will block `app:rendered`...

```suggestion
      nuxtApp.hook('app:rendered', () => { nextTick(checkIfLayoutUsed) })
```

Reply:
Makes sense 👍